images/server/install-packages.sh

```diff
@@ -12,8 +12,13 @@ need_curl() {
 
 get_custom_repos() {
     if [[ -n $1 ]]; then
+        urls=$1
         need_curl
-        for url in $1; do
+        if [[ $(wc -w <<< "$1") -eq 1 || $install_custom_ceph -ne 1 ]]; then
+            get_distro_ceph_repo
+            urls=${1%% *}
```

Comment:
What's this line do?

Reply:
This won't work if, for some reason, the string starts with a space. Why not use bash arrays ?

```bash
urls=(${1})
...
if [[ ${#urls[@]} -eq 1 || $ceph_from_custome -ne 1 ]]; then
    ...
    urls=(${urls[0]})
fi
for url in "${urls[@]}"; do
...
```